app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
```

Comment:
Definitely.  One of the things that bugged me was the colors but I couldn't think of a good solution so I was just gonna leave it to what you thought would look good.